lib/bundler/cli/config.rb

```diff
@@ -21,6 +21,8 @@ def run
         return
       end
 
+      return Bundler.ui.info(Bundler.settings[name]) if options[:parseable]
```

Comment:
Did you mean doing something like:

```
$ bundle config --local parseable true
$ bundle config --global parseable true
```

I tried, this line of change, does not affect original functionality.